framework/contrib/ice_updater/include/Post.h

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
```

Comment:
Same with this one... we might use it in other instances... it's not ICE specific...

Maybe something like `contrib/url_tools` or something?